Makefile

```diff
@@ -30,13 +33,24 @@ tag_release:
 	git commit -sm "android: bump version code" android/build.gradle
 	git tag -a "$(VERSION_LONG)"
 
+$(TOOLCHAINDIR):
```

Comment:
does this handle resuming if the mkdir succeeds but the tar is interrupted?

Reply:
Done.

It always spends the time to compute the sum of the toolchain even for make targets which don't use the toolchain, because I cannot get the in-target conditional to work. This makes tag_release take longer that it otherwise might, which we do once per release.